functionary/ui/views/builds.py

```diff
@@ -1,20 +1,19 @@
 from builder.models import Build
 
+from .generic import PermissionedDetailView, PermissionedListView
 from .tasks import FINISHED_STATUS
-from .view_base import (
-    PermissionedEnvironmentDetailView,
-    PermissionedEnvironmentListView,
-)
 
 
-class BuildListView(PermissionedEnvironmentListView):
+class BuildListView(PermissionedListView):
     model = Build
-    order_by_fields = ["-created_at"]
+    permissioned_model = "Package"
+    ordering = ["-created_at"]
```

Comment:
We had previously duplicated the buillt-in "ordering" property under the name "order_by_fields".  I switched everything over to using the built-in name so that I didn't have to reproduce the custom handling for it.